drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -16,8 +16,22 @@ using namespace Eigen;
 int main(int argc, char* argv[]) {
   const size_t num_lidar_points = 100;
 
+  // Get the final time of the simulation
   double final_time = argc >= 2 ? atof(argv[1]) : std::numeric_limits<double>::infinity();
-  cout << "Running simulation for " << final_time << " seconds." << endl;
+
+  // Get the maximum delay in seconds of the simulation time relative to the real time
+  double max_delay = argc >= 3 ? atof(argv[2]) : 1.0;
```

Comment:
The "1.0" here duplicates the default value from `Simulation.h`, which makes it difficult / brittle to keep them in sync.
Using `SimulationOptions().timeout_seconds` would be one fix, or just declare `SimulationOptions options;` earlier in the file.